### DISCUSSION TITLE
Data dumps

### DISCUSSION THREAD

**Discussion by captenmasin** [marcreichel/igdb-laravel#94] [Ideas]
The IGDB API has [Data Dumps](https://api-docs.igdb.com/#data-dumps) available for data partners, I wonder if this could be implemented into this package?

I haven't played around with the CSVs too much yet but maybe fetching models could search through a downloaded CSV before being fetched from the actual API

**Comment by marcreichel** [comment]
Looks interesting but as I do not have access to those data dumps I am not capable of implementing this feature.
